README.md

```diff
@@ -25,6 +25,7 @@ The following ROS 1 packages are required to build and use the bridge:
 * `std_msgs`
 * as well as the Python package `rospkg`
 
+**Note:** Bridge will connect only services with the same package name and service name so make sure that they are available when you build ros1_bridge.
```

Comment:
For messages the bridge allows custom mappings to be configured. Why should that not be the same for services?